server/grpc/api/event.go

```diff
@@ -44,9 +44,8 @@ func (s *EventServer) Create(ctx context.Context, req *api.CreateEventRequest) (
 	if err := json.Unmarshal([]byte(req.Event.Data), &data); err != nil {
 		return nil, fmt.Errorf("create event: data %s", err)
 	}
-
-	event, err := s.sdk.Event.Emit(instanceHash, req.Event.Key, data)
-	if err != nil {
+	event := event.Create(instanceHash, req.Event.Key, data)
+	if err := s.sdk.Event.Create(event); err != nil {
```

Comment:
This is weird stuff and one of the reasons why I don't like this pattern to pass the resource when we actually want to create the resource. We call 2 `event.create` this is so confusing